docs/getting-started/intro-to-dockstore-tools-and-workflows.rst

```diff
@@ -0,0 +1,88 @@
+| Nextflow               | - N/A                                    | - Any valid Nextflow workflow                   |
++------------------------+------------------------------------------+-------------------------------------------------+
+| Galaxy                 | - N/A*                                   | - Any valid Galaxy workflow                     |
++------------------------+------------------------------------------+-------------------------------------------------+
+| Versioning             | - Based off of image's tags              | - Based off of branches/tags from Git repository|
++------------------------+------------------------------------------+-------------------------------------------------+
+
+\* There are tools that make up Galaxy workflows from the Galaxy toolbox or ToolShed.
+Dockstore does not support registration of these tools.
+
+
+
+Tools
+-----
+
+Dockstore tool registration is meant for users who have created or have access/permissions to a Docker image registered to one of our supported container registries, and have
+written tool descriptor files (in CWL/WDL) that use it. At a basic level, the Docker image describes the tool environment and the descriptor files describe how the tool is run.
+If you are unfamiliar with Docker or how to write descriptor files, check out the following tutorials:
+
+- :doc:`Docker <getting-started-with-docker>`
+- :doc:`CWL <getting-started-with-cwl>`
+- :doc:`WDL <getting-started-with-wdl>`
+
+Dockstore:
+
+- has varying levels of support for images registered on Quay.io, DockerHub, GitLab, Amazon ECR, and Seven Bridges
+- supports descriptor files hosted on GitHub, BitBucket, GitLab, or written on Dockstore
+- supports descriptor files written in CWL or WDL
+- offers three different tool registration paths
+
+The most convenient way to register your tool and manage it on Dockstore is to have your image registered on Quay.io, your descriptor files hosted on GitHub, and choose our quick registration path.
+This gives Docktore the ability to automatically recognize the image's tags on Quay, link them back to the appropriate version on GitHub, and create the existing versions for you on Dockstore once you hit "Refresh".
+If your image is registered on one of our other supported registries, you will have to register your tool manually. This means each version you want put on Dockstore must be added manually.
+To learn more about our different registration options, read the following tutorials:
+
+- :doc:`Tools <dockstore-tools>`
+- :doc:`Hosted Tools <hosted-tools-and-workflows>`
+
+.. note::
+  Dockstore tool versions are based on the image's tags, not the tags/branches from the git repository where the descriptor files are hosted.
+
+
+Workflows
+---------
+
+Dockstore workflow registration is meant for users who have created or have access to descriptor files (in CWL, WDL, Galaxy, Nextflow). As mentioned above in the Tools section,
```

Comment:
`or have access to descriptor files` -> the descriptor files in git technically (a local copy won't help you). But you cover it later and you'd have to do a bit of rewording, so you can leave it.